src/sidebar/components/test/svg-icon-test.js

```diff
@@ -1,26 +1,20 @@
 'use strict';
 
-const angular = require('angular');
+const { createElement, render } = require('preact');
 
-const util = require('../../directive/test/util');
+const SvgIcon = require('../svg-icon');
 
-describe('svgIcon', function() {
-  before(function() {
-    angular.module('app', []).component('svgIcon', require('../svg-icon'));
+describe('SvgIcon', () => {
+  it("sets the element's content to the content of the SVG", () => {
+    const container = document.createElement('div');
+    render(<SvgIcon name="refresh" />, container);
+    assert.ok(container.querySelector('svg'));
```

Comment:
These tests use the DOM APIs to check for the `<svg>` element rather than using Enzyme (as our other React tests do) because content rendered via `dangerouslySetInnerHTML` is not part of the component tree that Enzyme sees.

Reply:
Very helpful—seems like a comment might be in order here so that this wisdom is not lost!